other-components.yml

```diff
@@ -5,16 +5,16 @@
 # 'services' tag.
 services:
     mender-artifact:
-        image: mendersoftware/mender-artifact:3.10.1
+        image: mendersoftware/mender-artifact:3.10.2
```

Comment:
I cannot pull this image:
```bash
# docker pull mendersoftware/mender-artifact:3.10.2
Error response from daemon: pull access denied for mendersoftware/mender-artifact, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```
should it be like that?

Reply:
These are not Docker images (I know, I know).
It is just a way to express versions.